hscontrol/handlers.go

```diff
@@ -138,34 +141,39 @@ type registerWebAPITemplateConfig struct {
 	Key string
 }
 
+var codeStyleRegisterWebAPI = styles.Props{
+	styles.Display:         "block",
+	styles.Padding:         "20px",
+	styles.Border:          "1px solid #bbb",
+	styles.BackgroundColor: "#eee",
+}
+
+var htmlRegisterWebAPI = elem.Html(nil,
+	elem.Head(
+		nil,
+		elem.Title(
+			nil,
+			elem.Text("Registration - Headscale"),
+		),
+		elem.Meta(attrs.Props{
+			attrs.Name:    "viewport",
+			attrs.Content: "width=device-width, initial-scale=1",
+		},
+		),
+		elem.Body(attrs.Props{
+			attrs.Style: styles.Props{
+				styles.FontFamily: "sans",
+			}.ToInline(),
+		},
+			elem.H1(nil, elem.Text("headscale")),
+			elem.H2(nil, elem.Text("Machine registration")),
+			elem.P(nil, elem.Text("Run the command below in the headscale server to add this machine to your network:")),
+			elem.Code(attrs.Props{attrs.Style: codeStyleRegisterWebAPI.ToInline()},
+				elem.Raw(`headscale nodes register --user USERNAME --key {{.Key}}`)),
```

Comment:
Lets drop the use of the templates, this is the only thing needing it, so you can make this whole thing a function, and then pass a `key string` argument, which is then used here
I think it would be something like:
```suggestion
				elem.Text(fmt.Sprintf("headscale nodes register --user USERNAME --key %s", key)),
```